src/demo-app/select/select-demo.html

```diff
@@ -6,7 +6,7 @@
   <md-card>
     <md-card-subtitle>ngModel</md-card-subtitle>
 
-    <md-select placeholder="Drink" [color]="drinksTheme" [(ngModel)]="currentDrink" [required]="drinksRequired"
+    <md-select placeholder="Drink" [color]="drinksTheme" [(ngModel)]="currentDrink" [required]="drinksRequired && !hideRequiredMarker"
```

Comment:
This is just making the select not required, we want to set `[hideRequiredMarker]="hireRequiredMarker"`, that way its still required but we just don't show the star